booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java

```diff
@@ -78,101 +78,131 @@ private BookingScenarioListBuilder thenAllPathsFrom(BookingState bookingState) {
   private BookingScenarioListBuilder thenAllPathsFrom(
       BookingState bookingState, BookingState originalBookingState) {
     return switch (bookingState) {
-      case CANCELLED, COMPLETED, DECLINED, REJECTED  -> then(shipper_GetBooking(bookingState));
+      case COMPLETED  -> then(
+        shipper_GetBooking(bookingState).thenEither(
+         uc7_shipper_submitBookingAmendment(true)
+            .then(shipper_GetBooking(bookingState))
+        )
```

Comment:
Any reason why `cancel` is not being tried here as well?

Reply:
As per the story user performs UC7 when Booking status is COMPLETED and it hasn't been defined in the story.

Reply:
Challenge the story then to confirm whether this omission is intentional or an oversight